server.js

```diff
@@ -124,7 +124,7 @@ if (fs.existsSync(whitelistPath)) {
 }
 
 const whitelistMode = getConfigValue('whitelistMode', true);
-const autorun = getConfigValue('autorun', false) && cliArguments.autorun !== false && !cliArguments.ssl;
+const autorun = getConfigValue('autorun', false) && cliArguments.autorun && !cliArguments.ssl;
```

Comment:
> but it's still false in the code to fallback to no autorun in case it is removed from yaml
Couldn't we just have the default config value for autorun be `true` then?

Reply:
The reason `getConfigValue('autorun', false)` returns true even if you remove it from your config is that it falls back to the value in `default/config.yaml`, which is `true`.